### PR TITLE
Documentation Overhaul

### PR DESCRIPTION
**Pull request by GeorgeR227**
All documentation pages are being reviewed and improved to clean up our pages and to cut down on the documentation building time.

### CODE REVIEW

**Review by jpfairbanks — COMMENTED** [review]
Great work @GeorgeR227, this is almost done. I added some inline comments.

File: docs/src/budyko_sellers_halfar.md [hunk 1]
```diff
@@ -105,7 +113,8 @@ budyko_sellers_composition_diagram = @relation () begin
   diffusion(Tₛ, HT, cosϕᵖ)
   insolation(Q, cosϕᵖ)
 end
-to_graphviz(budyko_sellers_composition_diagram, box_labels=:name, junction_labels=:variable, prog="circo")
+
+to_graphviz(budyko_sellers_composition_diagram, box_labels=:name, junction_labels=:variable, prog="circo") # TODO: Make this into a helper
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:117]: Yeah that sounds good, and I can place it in that ```docinfo.jl``` file I added.

File: docs/src/ice_dynamics.md [hunk 2]
```diff
@@ -152,12 +161,28 @@ constants_and_parameters = (
...
+      append!(I, [e,e])
+      append!(V, [0.5, 0.5])
+  end
+  avg_mat = sparse(I,J,V)
+end
+
+# TODO: Move this sharp matrix out of the generate
+function create_sharp_matrix(sd)
+
+end
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:184]: It's not done and I left the empty stub as a reminder. The idea is to refactor this code out into the function stub I created since this is all being run every sharp computation. https://github.com/AlgebraicJulia/Decapodes.jl/blob/85c70f1a3a837a68b9a5d2fd9b6f67640337cc07/docs/src/ice_dynamics.md?plain=1#L193-L204 If the doc building times are quick enough we could just come back to this when we have a real implementation of this 1D sharp rather than this bespoke one.

File: docs/src/ice_dynamics.md [hunk 3]
```diff
@@ -178,55 +203,37 @@ function generate(sd, my_symbol; hodge=GeometricHodge())
...
-    end
-    :^ => (x,y) -> x .^ y
-    :* => (x,y) -> x .* y
-    :show => x -> begin
-      @show x
-      x
+    :mag => x -> norm.(x)
+    :avg₀₁ => begin
+      avg_mat = create_average_matrix(sd)
+      x -> avg_mat * x
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:209]: No, same as before we need that function.

File: docs/src/klausmeier/klausmeier.md [hunk 4]
```diff
@@ -21,23 +26,24 @@ The lofty goal of this document, and of Decapodes itself, is that through both e
...
-using CairoMakie
+using GeometryBasics: Point2
 using JLD2
 using LinearAlgebra
 using MLStyle
-using ComponentArrays
 using OrdinaryDiffEq
-using GeometryBasics: Point2
 Point2D = Point2{Float64}
+nothing # hide
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/klausmeier/klausmeier.md:46]: This is just hiding the return value. I like it cause it reduces noise in the resulting docs page.

File: docs/src/ice_dynamics.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jpfairbanks) [reply docs/src/ice_dynamics.md:184]: If refactoring this will take longer than *today*, then I think we can merge this and you can open a new issue for this refactor to create a permanent 1D sharp. Just reading the loops, I don't like this implementation. I think there is a much more performant way to do build this matrix that is worth figuring out.

[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:184]: Yeah that sounds good to me. If I remember correctly these docs didn't take too long to build so chances are I'll leave this alone to be refactored properly later.

File: docs/src/nhs/nhs.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Implement Oceananigans.jl's NonhydrostaticModel in the Discrete Exterior Calculus
 
-Let's use Decapodes to implement the [NonhydrostaticModel](https://clima.github.io/OceananigansDocumentation/stable/physics/nonhydrostatic_model/) from Oceananigans.jl. We will take the opportunity to demonstrate how we can use our "algebra of model compositions" to encode certain guarantees on the models we generate. We will use the [2D Turbulence](https://clima.github.io/OceananigansDocumentation/stable/generated/two_dimensional_turbulence/) as a guiding example, and use only equations found in the Oceananigans docs to construct our model.
+Let's use Decapodes to implement the [NonhydrostaticModel](https://clima.github.io/OceananigansDocumentation/stable/physics/nonhydrostatic_model/) from Oceananigans.jl. We will take the opportunity to demonstrate how we can use our "algebra of model compositions" to encode certain guarantees on the models we generate. We will use the [2D Turbulence](https://clima.github.io/OceananigansDocumentation/stable/literated/two_dimensional_turbulence/) as a guiding example, and use only equations found in the Oceananigans docs to construct our model.
```
Comment (GeorgeR227) [inline docs/src/nhs/nhs.md:3]: @lukem12345 The previous link to 2D turbulence was broken. Can you confirm this new link is correct?

File: docs/src/ice_dynamics.md [hunk 6]
```diff
@@ -56,7 +61,8 @@ end
 to_graphviz(halfar_eq2)
 ```
 
-And here, a formulation of Glen's law from J.W. Glen's 1958 "The flow law of ice".
+And here, a formulation of Glen's law from J.W. Glen's 1958 "The flow law of ice". <!-- TODO: Add a link for this too -->
```
Comment (GeorgeR227) [inline docs/src/ice_dynamics.md:64]: @lukem12345 Can we also get a link to this paper?

**Comment by lukem12345** [comment]
http://go.owu.edu/~chjackso/Climate/papers/Glen_1958_The%20flow%20law%20of%20ice.pdf

**Comment by GeorgeR227** [comment]
Benchmark times as of this last commit. These are built locally, not on HiPerGator like in https://github.com/AlgebraicJulia/Decapodes.jl/pull/231#issuecomment-2135893834.

navier_stokes/ns - 0 seconds
halmo - removed
overview - 30 seconds
klausmeier - 24 seconds
ice_dynamics - 42 seconds
grigoriev - 21 seconds
budyko_sellers_halfar - 25 seconds
cism - 27 seconds
nhs - 11 seconds (lite version)
equations - 1 second
bc_debug - 22 seconds
poiseuille - 9 seconds
cahn-hilliard - 165 seconds

File: docs/src/budyko_sellers_halfar.md [hunk 7]
```diff
@@ -338,50 +346,58 @@ We can save the solution file to examine later.
...
 # Initial frame
 frames = 100
-fig = Figure(; size = (800, 800))
+fig = Figure()
 ax1 = CairoMakie.Axis(fig[1,1])
-xlims!(ax1, extrema(map(x -> x[1], point(s′))))
+xlims!(ax1, extrema(map(x -> x[1], point(s))))
 ylims!(ax1, extrema(soln(tₑ).Tₛ))
 Label(fig[1,1,Top()], "Surface temperature, Tₛ, [C°]")
-Label(fig[2,1,Top()], "Line plot of temperature from North to South pole, every $(tₑ/frames) time units")
```
Comment (lukem12345) [inline docs/src/budyko_sellers_halfar.md:360]: Why is this removed

File: docs/src/budyko_sellers_halfar.md [hunk 8]
```diff
@@ -338,50 +346,58 @@ We can save the solution file to examine later.
...
 
 # Animation
 record(fig, "budyko_sellers_halfar_T.gif", range(0.0, tₑ; length=frames); framerate = 15) do t
-  lines!(fig[1,1], map(x -> x[1], point(s′)), soln(t).Tₛ)
+  lines!(fig[1,1], map(x -> x[1], point(s)), soln(t).Tₛ)
 end
 end
-
+# TODO: There seems to be some instability here
```
Comment (lukem12345) [inline docs/src/budyko_sellers_halfar.md:376]: It's ok

File: docs/src/ch/cahn-hilliard.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+end
+
+to_graphviz(CahnHilliard)
+```
+
+In this equation `C` will represent the concentration of the binary fluid, ranging from `-1` to `1` to differentiate between different phases. We also have a diffusion constant `D` and a constant `γ` whose square root is the length of the transition regions. This formulation of the Cahn-Hilliard equation was drawn from the Wikipedia page on the topic found [here](https://en.wikipedia.org/wiki/Cahn%E2%80%93Hilliard_equation).
+
+## Loading the Data
+
+We now generate the mesh information. We'll run the equation on a triangulated grid. I hide the mesh visualization code for clarity.
```
Comment (lukem12345) [inline docs/src/ch/cahn-hilliard.md:51]: I -> we

File: docs/src/cism/cism.md [hunk 10]
```diff
@@ -1,41 +1,44 @@
...
+```
+
 The Decapodes framework takes high-level representations of physics equations and automatically generates solvers.
 
 We do so by translating equations from vector calculus notation to the "discrete exterior calculus" (DEC). This process is roughly about recognizing whether physical quantities represent scalar or vector quantities, and recognizing whether differential operators represent gradient, divergence, and so on.
 
-In this benchmark, we will implement the "small slope approximation" of glacial dynamics used by P. Halfar in his 1981 work "On the dynamics of the ice sheets" by taking his original formulation, translating it into the DEC, then providing a mesh and initial conditions.
-
-The initial conditions used here are exactly those considered by W. H. Lipscomb et al. in "Description And Evaluation of the Community Ice Sheet Model (CISM) v2.1" (2019).
+In this benchmark, we will implement the "small slope approximation" of glacial dynamics used by P. Halfar in his 1981 work "On the dynamics of the ice sheets" by taking his original formulation, translating it into the DEC, then providing a mesh and initial conditions. <!-- TODO: Link to paper-->
```
[the remark this answers is not in the document]
Comment (lukem12345) [reply docs/src/cism/cism.md:12]: https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/JC086iC11p11065

File: docs/src/cism/cism.md [hunk 11]
```diff
@@ -327,7 +329,7 @@ h_diff = soln(tₑ).dynamics_h - hₐ
 maximum(abs.(h_diff))
 ```
 
-We compute root meand square error (RMSE) as well, both over the entire domain, and *excluding where the ice distribution is 0 in the analytic solution.* (Considering the entire domain decreases the RMSE, while not telling you much about the dynamics in the area of interest.) Note that the official CISM benchmark reports 6.43 and 9.06 RMSE for their two solver implementations.
+We compute root-mean-square error (RMSE) as well, both over the entire domain, and *excluding where the ice distribution is 0 in the analytic solution.* This is done since considering the entire domain decreases the RMSE while not telling you much about the area of interest. Note that the official CISM benchmark reports `6.43` and `9.06` RMSE for their two solver implementations.
```
Comment (lukem12345) [inline docs/src/cism/cism.md:332]: I liked the sentence better before

File: docs/src/grigoriev/grigoriev.md [hunk 12]
```diff
@@ -1,44 +1,49 @@
 # Halfar's model of glacial flow
 
-Let's model glacial flow using a model of how ice height of a glacial sheet changes over time, from P. Halfar's 1981 paper: "On the dynamics of the ice sheets".
+```@setup INFO
+include(joinpath(Base.@__DIR__, ".." , "..", "docinfo.jl"))
+info = DocInfo.Info()
+```
+
+Let's model glacial flow using a model of how ice height of a glacial sheet changes over time, from P. Halfar's 1981 paper: "On the dynamics of the ice sheets". <!--- TODO: Need a link for this -->
```
Comment (lukem12345) [inline docs/src/grigoriev/grigoriev.md:8]: https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/JC086iC11p11065

File: docs/src/grigoriev/grigoriev.md [hunk 13]
```diff
@@ -1,44 +1,49 @@
...
+
+The ice thickness data is stored in a TIF that can be downloaded [here](https://zenodo.org/api/records/7735970/files-archive). We have downloaded it locally, and load it using basic `FileIO`.
 
 ``` @example DEC
 file_name = "Icethickness_Grigoriev_ice_cap_2021.tif"
 ice_thickness_tif = load(file_name)
 ```
 
-This data may appear to be a simple binary mask, but that is only because values with no ice are set to `-Inf`. We will account for this we interpolate our data.
+This data may visually appear to be a binary mask but that is only because values with no ice are set to `-Inf`. We will account for this we interpolate our data.
```
Comment (lukem12345) [inline docs/src/grigoriev/grigoriev.md:46]: when we interpolate

File: docs/src/grigoriev/grigoriev.md [hunk 14]
```diff
@@ -63,47 +69,46 @@ Let's generate a triangulated grid located at the appropriate coordinates:
...
-subdivide_duals!(s, Barycenter())
+s[:point] = map(x -> x + Point3D(MIN_X, MIN_Y, 0), s[:point])
+sd = EmbeddedDeltaDualComplex2D{Bool, Float64, Point3D}(s)
+subdivide_duals!(sd, Barycenter())
 
 fig = Figure()
 ax = CairoMakie.Axis(fig[1,1])
-wf = wireframe!(ax, s)
+wf = wireframe!(ax, sd)
 display(fig)
```
Comment (lukem12345) [inline docs/src/grigoriev/grigoriev.md:84]: This is not rendering

File: docs/src/grigoriev/grigoriev.md [hunk 15]
```diff
@@ -63,47 +69,46 @@ Let's generate a triangulated grid located at the appropriate coordinates:
...
 end
 
 # Store these values to be passed to the solver.
 u₀ = ComponentArray(h=h₀, stress_A=A)
-constants_and_parameters = (
-  n = n,
-  stress_ρ = ρ,
-  stress_g = g,
-  stress_A = A)
+constants_and_parameters = (n = n, stress_ρ = ρ,
```
Comment (lukem12345) [inline docs/src/grigoriev/grigoriev.md:104]: This was ok before

File: docs/src/grigoriev/grigoriev.md [hunk 16]
```diff
@@ -63,47 +69,46 @@ Let's generate a triangulated grid located at the appropriate coordinates:
...
+                            stress_g = g, stress_A = A)
+nothing # hide
 ```
 
-# Defining and Composing Models
-For exposition on this Halfar Decapode, see our [Glacial Flow](https://algebraicjulia.github.io/Decapodes.jl/dev/ice_dynamics) docs page. You can skip ahead to the next section.
+## Defining and Composing Models
+
+For exposition on this Halfar Decapode, see our [Glacial Flow](../ice_dynamics.md) docs page. Otherwise, you may skip ahead to the next section.
```
Comment (lukem12345) [inline docs/src/grigoriev/grigoriev.md:111]: Does Documenter replace this with a link to the render docs page? If not put the url back

File: docs/src/ice_dynamics.md [hunk 17]
```diff
@@ -69,9 +75,9 @@ end
 to_graphviz(glens_law)
 ```
 
-# Composing models
+## Composing models
 
-We can use operadic composition to specify how our models come together. In this example, we have two Decapodes, and two quantities that are shared between them.
```
Comment (lukem12345) [inline docs/src/ice_dynamics.md:74]: This needs to go back

File: docs/src/ice_dynamics.md [hunk 18]
```diff
@@ -105,22 +111,25 @@ resolve_overloads!(ice_dynamics1D, op1_res_rules_1D, op2_res_rules_1D)
...
 
-# Define a mesh
+## Define a mesh
 
 We'll need a mesh to simulate on. Since this is a 1D mesh, we can go ahead and make one right now:
 
 ``` @example DEC
-# This is a 1D mesh, consisting of edges and vertices.
-s′ = EmbeddedDeltaSet1D{Bool, Point2D}()
+# This is an empty 1D mesh.
```
Comment (lukem12345) [inline docs/src/ice_dynamics.md:119]: The previous comment needs to go back

File: docs/docinfo.jl [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+
+function get_report(info::Info)
+  info.finish_time = now()
+  elapsed = get_elapsed(info)
+  elapsed_sec = round(elapsed, Dates.Second(1))
+  @info "Page built in $(elapsed_sec)."
+  @info "This page was last built at $(info.finish_time)."
+end
+
+draw(deca) = to_graphviz(deca, box_labels=:name, junction_labels=:variable, prog="circo")
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/docinfo.jl:26]: I agree but why is this the way things are.

File: docs/src/bc_debug.md [hunk 20]
```diff
@@ -101,16 +100,15 @@ boundary conditions and so we will use the `plot_mesh` from the previous
 example instead of the mesh with periodic boundary conditions. Because the mesh
 is only a primal mesh, we also generate and subdivide the dual mesh.
 
-`Rectangle_30x10` is a default mesh that is downloaded via `Artifacts.jl` when a user installs Decapodes. Via CombinatorialSpaces.jl, we can instantiate any `.obj` file of triangulated faces as a simplicial set.
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/src/bc_debug.md:104]: Because this isn't the proper time to bring this up. This is one of the final doc pages. This is done instead in `overview.md`

File: docs/docinfo.jl (hunk 19 above)
[the remark this answers is not in the document]
Comment (lukem12345) [reply docs/docinfo.jl:26]: DW - Decapodes split removed the precompilation bottleneck for some teams by removing numerical simulation dependencies. Smooths way for further DSLs with ACSet and MLStyle support in the AJ ecosystem.

File: docs/src/budyko_sellers_halfar.md (hunk 7 above)
Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:360]: It was making the plots show up with a lot of white space at the bottom. You can check it out here: https://algebraicjulia.github.io/Decapodes.jl/dev/budyko_sellers_halfar/

File: docs/src/ch/cahn-hilliard.md (hunk 9 above)
Comment (GeorgeR227) [reply docs/src/ch/cahn-hilliard.md:51]: Why, every other page uses we and not I.

Comment (GeorgeR227) [reply docs/src/ch/cahn-hilliard.md:51]: See: https://github.com/AlgebraicJulia/Decapodes.jl/blob/7a9846bcb722c8c4150612e31e6b500a4a3469f2/docs/src/overview.md?plain=1#L105-L107

File: docs/src/cism/cism.md (hunk 11 above)
Comment (GeorgeR227) [reply docs/src/cism/cism.md:332]: The use of a parentheses in a sentence is to include an idea which is not essential. I argue that this is explaining why we compute RMSE the way we do and so is definitely essential.

File: docs/src/grigoriev/grigoriev.md (hunk 16 above)
Comment (GeorgeR227) [reply docs/src/grigoriev/grigoriev.md:111]: Yes it does, and we should definitely be using relative links.

File: docs/src/ch/cahn-hilliard.md (hunk 9 above)
Comment (lukem12345) [reply docs/src/ch/cahn-hilliard.md:51]: I know. The last sentence here says I

File: docs/src/ice_dynamics.md (hunk 17 above)
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:74]: In that case, `operadic composition` should be better explained or at least have a link to a resource describing it. This term is not a common one nor do we define it here ourselves.

File: docs/src/ice_dynamics.md (hunk 18 above)
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:119]: But this next line is an empty 1D mesh.

Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:119]: The comment after correctly describes the primal mesh.

File: docs/src/budyko_sellers_halfar.md (hunk 7 above)
Comment (lukem12345) [reply docs/src/budyko_sellers_halfar.md:360]: What happens when you change the 800,800 to a rectangle

File: docs/docinfo.jl (hunk 19 above)
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/docinfo.jl:26]: Yes but that numerical simulation dependency is removed in #216. In anyway I'd rather not tie this entire PR to a separate PR in another package over a little issue like this.

File: docs/src/ch/cahn-hilliard.md (hunk 9 above)
Comment (GeorgeR227) [reply docs/src/ch/cahn-hilliard.md:51]: Oh, I misunderstood. Fixed.

File: docs/docinfo.jl (hunk 19 above)
[the remark this answers is not in the document]
Comment (lukem12345) [reply docs/docinfo.jl:26]: Commit to main on DW

File: docs/src/grigoriev/grigoriev.md (hunk 15 above)
Comment (GeorgeR227) [reply docs/src/grigoriev/grigoriev.md:104]: Sure, fixed.

File: docs/src/ice_dynamics.md (hunk 17 above)
Comment (lukem12345) [reply docs/src/ice_dynamics.md:74]: Ok, link to Sophie's paper

File: docs/docinfo.jl (hunk 19 above)
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/docinfo.jl:26]: We might also have to tag a new release for that to work.

File: docs/src/budyko_sellers_halfar.md (hunk 7 above)
Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:360]: I tried and it's the same. The problem is that another entire portion of the figure is made but is only filled with some text.

File: docs/src/grigoriev/grigoriev.md (hunk 13 above)
Comment (GeorgeR227) [reply docs/src/grigoriev/grigoriev.md:46]: Fixed.

File: docs/src/grigoriev/grigoriev.md (hunk 14 above)
Comment (GeorgeR227) [reply docs/src/grigoriev/grigoriev.md:84]: Yeah you're right, the problem was from even before. I'll replace this with the image save - markdown display trick.

File: docs/src/ice_dynamics.md (hunk 17 above)
Comment (GeorgeR227) [reply docs/src/ice_dynamics.md:74]: This one? https://arxiv.org/abs/2105.12282

**Comment by lukem12345** [comment]
Yeah that's the good one to use

File: docs/src/grigoriev/grigoriev.md (hunk 14 above)
Comment (GeorgeR227) [reply docs/src/grigoriev/grigoriev.md]: This generates correctly now.

File: docs/make.jl [hunk 21]
```diff
@@ -43,24 +42,23 @@ makedocs(
...
     "Glacial Flow" => "ice_dynamics.md",
-    "Grigoriev Ice Cap" => "grigoriev.md",
+    "Grigoriev Ice Cap" => "grigoriev/grigoriev.md",
     "Budyko-Sellers-Halfar" => "budyko_sellers_halfar.md",
-    "CISM v2.1" => "cism.md",
-    "NHS" => "nhs.md",
-    "Equations" => "equations.md",
-    "ASCII Operators" => "ascii.md",
-    "Misc Features" => "bc_debug.md",
+    # "Halfar-NS" => "halmo.md", # TODO: Remove since results are not interesting?
```
[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/make.jl]: I added back Halmo and did a quick sweep through. Main point I don't like is use of ```default_dec_matrix_generate``` which is bad behavior but to be fair is not the only one to do that.

[the remark this answers is not in the document]
Comment (GeorgeR227) [reply docs/make.jl]: Also refactoring that out now would be a headache and not worth it.

**Comment by GeorgeR227** [comment]
Adding to the benchmarks at https://github.com/AlgebraicJulia/Decapodes.jl/pull/231#issuecomment-2142895966:
halmo - 54 seconds

File: docs/src/budyko_sellers_halfar.md (hunk 7 above)
Comment (jpfairbanks) [reply docs/src/budyko_sellers_halfar.md:360]: I think the problem is `Label(fig[2,1,Top()]` should be `Label(fig[1,1,Bottom()]`

**Comment by lukem12345** [comment]
Theory: There is a race condition.

The issue occurs (sometimes) when we generate a `.jl` file. Perhaps Documenter is seeing the `.jl` file and doing its Documenter black magic on it before the file is finished writing etc. etc.

**Comment by GeorgeR227** [comment]
Well the ```.jl``` file is removed and it's still dead-locking. Leading theory for me right now is that it is something to do with the ```save_vorticity``` function.

**Comment by GeorgeR227** [comment]
I'm not sure what to say but it seems like ```save_vorticity``` may have been the cause. I've run the docs multiple times now and it seems to work.

**Comment by GeorgeR227** [comment]
Pretty sure the issue occurs when we actually create the gif. Still not sure as to why.
image

**Comment by lukem12345** [comment]
Interesting. May be worth your while to look at `top` and see when `ffmpeg`
spawns and terminates

File: docs/src/budyko_sellers_halfar.md (hunk 7 above)
Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:360]: I tried this but now it just overlaps with the bottom axis:
image

Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:360]: I'm going to try messing around with the size of the figure but if that doesn't work I'll leave it as is now.

Comment (GeorgeR227) [reply docs/src/budyko_sellers_halfar.md:360]: Ok got it, we just had to use `xlabel` on the axis.

**Comment by GeorgeR227** [comment]
@lukem12345 You were right about `ffmpeg.exe` this seems to be deadlocked and is the only program with this handle.
image

**Comment by GeorgeR227** [comment]
I've opted to just remove the offending code in `halmo.md` because I fear the hanging issue is not something we really have the ability to fix. I also still don't know the root cause of the hanging.

File: src/canon/Physics.jl [hunk 22]
```diff
@@ -157,7 +157,7 @@ end)
 )
 
 @docapode("Poiseuille Density"
```
Comment (GeorgeR227) [inline src/canon/Physics.jl:159]: @lukem12345 @quffaro This link appears to be dead and there's no Wikipedia page on this specific topic. Can you provide a fresh link?

Comment (GeorgeR227) [reply src/canon/Physics.jl:159]: This is the old link: https://en.wikipedia.org/wiki/Hagen-poiseuille_density

Comment (jpfairbanks) [reply src/canon/Physics.jl:159]: https://en.wikipedia.org/wiki/Hagen%E2%80%93Poiseuille_equation

**Review by lukem12345 — APPROVED** [review]
